src/app/main.cpp

```diff
@@ -252,6 +252,11 @@ int main(int argc, char *argv[])
         signal(SIGSEGV, sigAbnormalHandler);
 #endif
 
+#ifdef Q_OS_MAC
+        // On OS X the standard is to not show icons in the menus
+        app->setAttribute(Qt::AA_DontShowIconsInMenus);
```

Comment:
there are already an `#if defined(Q_OS_MAC)` block at line 210
you should move it there.